python/gpufort/scanner/parser.py

```diff
@@ -629,6 +629,7 @@ def is_end_statement_(tokens, kind):
                     current_linemap["statements"]):
                 try:
                     expand_statement_functions_(current_statement)
+                    original_statement = current_statement["body"]
```

Comment:
Won't accept frontend changes now as I am currently changing it on a separate development branch.
I suggest to keep a local copy and not include to this PR.

Reply:
Please feel free to reject the changes. I have local copies.